### PR TITLE
Websocket connection to backend server & commands page

### PR DESCRIPTION
**Pull request by Deepak-Lalwani**
1. Instead of getting the order book data from database, now we are creating a websocket connection between client proxy server and backend (window) server.
2. Create commands page with initial changes (only command type and hft flag to be sent) and api structure for sending request and response via REST call to the backend server.

### CODE REVIEW

File: server/services/commands/utils.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  try{
+    const response = await axios.get(config.SERVERHOSTNAME, { params: req.query });
+    return response.data;
+  } catch (error) {
+    console.log("Error while calling the API : ", error);
+    return res.end({"success": false});
+  }
+}
+
+module.exports = { sendCommand }
```
Comment (pskucherov) [inline server/services/commands/utils.js]: add new line at end of the file for all files, where github mark red circle. Or use `eslint --fix filepath.js`

File: src/views/app/commands/index.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    super(props);
+
+    this.state = {
+      hftChecked: false,
+      commandType: this.commands[0].command,
+      showResult: false,
+      commandResult: null,
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+}
```
Comment (pskucherov) [inline src/views/app/commands/index.js]: use `eslint --fix src/views/app/commands/index.js`

File: src/views/app/commands/index.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    {
+      command: 'newstoporder',
+    },
+    {
+      command: 'newcondorder',
+    },
+    {
+      command: 'cancelorder',
+    },
+  ];
```
Comment (pskucherov) [inline src/views/app/commands/index.js]:
```
commands = [
    'server_status',
    'get_securities',
    ...etc...
].map(command => { return { command }; });
```

File: server/services/commands/utils.js (hunk 1 above)
Comment (Deepak-Lalwani) [reply server/services/commands/utils.js]: Fixed eslint errors.

File: src/views/app/commands/index.js (hunk 2 above)
Comment (Deepak-Lalwani) [reply src/views/app/commands/index.js]: Fixed eslint errors

File: src/views/app/commands/index.js (hunk 3 above)
Comment (Deepak-Lalwani) [reply src/views/app/commands/index.js]: Moved the commands object to a separate file in constant folder. Planning to keep the command as an object in an array to enable conditional rendering of fields.

**Pull request merged**